ktor-client/ktor-client-tests/build.gradle

```diff
@@ -2,6 +2,17 @@ description = 'Common tests for client'
 
 apply plugin: "kotlinx-serialization"
 
+class BackgroundJavaProcess extends JavaExec {
+    public Thread thread = new Thread({ super.exec() })
```

Comment:
Hope it doesn't trigger any computation/configuration on demand or etc in the separate thread